### PR TITLE
Create 2024-02-29-coordination.md

### PR DESCRIPTION
**Pull request by urania277**
Coordination minutes 2024/02/29. 

For @graeme-a-stewart: I'm not 100% sure what the best way it is to inform the coordination team (including WG conveners) on how to join the JENA computing WG lists, feel free to edit if you have more information than what I have added. I also didn't add the mailing list for WP2 to avoid potential spam.

### CODE REVIEW

**Comment by graeme-a-stewart** [comment]
For the JENA things there is now an official page: <https://nupecc.org/jenaa/index.php?display=computing>. I think we should link to that one.

File: organization/_posts/2024/2024-02-29-coordination.md [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+Submissions describing projects or other ideas, such as training initiatives or community support structures are also warmly welcomed.
+This is your chance to show off how good your project is and to meet and talk to current and future users!
+
+**Please broadcast this call widely and encourage contributors.** Deadline is 18 March.
+
+### LHCC
+
+Generators presented at the last LHCC meeting, with a number of comments/findings from the HSF discussions. Echoed from the generator workshop from last November: 
+- need better pipelines for generator software development
+- [see also workshop conclusions]
```
Comment (graeme-a-stewart) [inline organization/_posts/2024/2024-02-29-coordination.md:41]: Is there some link that should go here?

File: organization/_posts/2024/2024-02-29-coordination.md [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+- LHCC would like experiments to get involved in JENA activities and HSF could be the catalyser
+
+### JENAA
+
+[announcement](https://www.appec.org/news/joint-ecfa-nupecc-appec-activities-jenaa), [website](https://www.appec.org/implementation/joint-ecfa-nupecc-appec-activities), [2023 workshop](https://agenda.infn.it/event/34738/)
```
Comment (graeme-a-stewart) [inline organization/_posts/2024/2024-02-29-coordination.md:46]: Should add:

```
- [JENAA Computing Page](https://nupecc.org/jenaa/index.php?display=computing)
```

I think that can replace the other links.

File: organization/_posts/2024/2024-02-29-coordination.md [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+  - Saptaparna: connection to experiments? A: does not need to be too tight. 
+  - Not too late for students to join project proposals, broad advertising. 
+
+### HSF Evolution
+
+Work in progress: next meeting of the core coordination team next week, producing some text for everyone to look at. 
+
+## Working Group Updates
+
+### General Standing Reminders
```
Comment (graeme-a-stewart) [inline organization/_posts/2024/2024-02-29-coordination.md:105]: I always take these general standing reminders out of the published minutes - there's no sense in having them on the website every time

File: organization/_posts/2024/2024-02-29-coordination.md [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+
+We have the ability to put event banners on the HSF website. All it requires is a markdown file with the event data. See, e.g., files [here](https://github.com/HSF/hsf.github.io/tree/main/announcements/_posts/2023).
+
+#### Meetings
+
+Please try and book meetings in Indico at least 2 weeks in advance!
+
+That way they go into the calendar early and they will be included in the weekly email announcement that goes to HSF Forum.
+
+### Data Analysis
```
Comment (graeme-a-stewart) [inline organization/_posts/2024/2024-02-29-coordination.md:117]: Generally if there's an NTR then I just delete the item

**Review by graeme-a-stewart — CHANGES_REQUESTED** [review]
Hi @urania277 - thank so much for doing these and sorry for the very late review!

I would say it's usually better to make a branch in the main repository as then one can push fixes directly into it rather than asking you to make minor edits.

**Comment by urania277** [comment]
Aha, sorry for branching my fork, ATLAS habits! Will make the change shortly, let me know if anything else is needed as I can change it quickly

On Mar 15, 2024 08:55, Graeme A Stewart ***@***.***> wrote:

For the JENA things there is now an official page: https://nupecc.org/jenaa/index.php?display=computing. I think we should link to that one.

**Comment by graeme-a-stewart** [comment]
To be fair, we started also with the fork model, but then it became clear that it was easier just to work on the one repository.

Thanks!

File: organization/_posts/2024/2024-02-29-coordination.md (hunk 1 above)
Comment (urania277) [reply organization/_posts/2024/2024-02-29-coordination.md:41]: Hmm, I can't find the "November workshop" mentioned there on indico or on the generators WG page - it's been a long week so I may just be unable to search, but if @sextonkennedy has an idea of which workshop was mentioned here I'll add the link.

**Review by graeme-a-stewart — APPROVED** [review]

**Comment by graeme-a-stewart** [comment]
Thanks @urania277!

**Pull request merged**